test_triton_extension_backend.py shows an error:
> torch._dynamo.exc.BackendCompilerFailed: backend='inductor' raised:
> LoweringException: NotImplementedError: 
>  target: aten.min.default

This is due to changes in other parts of the code and this method should be removed.
See also https://github.com/pytorch/pytorch/pull/144399

Author: Alexander Grund (TU Dresden)

diff --git a/test/inductor/extension_backends/triton/device_interface.py b/test/inductor/extension_backends/triton/device_interface.py
index 9ca96e71a7d..14b3ca5436e 100644
--- a/test/inductor/extension_backends/triton/device_interface.py
+++ b/test/inductor/extension_backends/triton/device_interface.py
@@ -108,10 +108,6 @@ class DeviceInterface(device_interface.DeviceInterface):
     def synchronize(device) -> None:
         pass
 
-    @staticmethod
-    def get_device_properties(device) -> DeviceProperties:
-        raise NotImplementedError
-
     # Can be mock patched by @patch decorator.
     @staticmethod
     def is_available() -> bool:
